proto/proto.go

```diff
@@ -9,20 +7,23 @@ import (
 	"github.com/0xsequence/authcontrol/proto"
 )
 
+//go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=proto.ridl -target=golang@v0.15.0 -pkg=proto -server -client -out=./proto.gen.go
+//go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=proto.ridl -target=typescript@v0.12.0 -client -out=./clients/quotacontrol.gen.ts
```

Comment:
return these to top of the file please, and i think we don't need to generate server so we don't need `clients` folder.
```suggestion
//go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=proto.ridl -target=golang@v0.15.0 -pkg=proto -client -out=./proto.gen.go
//go:generate go run github.com/webrpc/webrpc/cmd/webrpc-gen -schema=proto.ridl -target=typescript@v0.12.0 -client -out=./quotacontrol.gen.ts
```

Reply:
We need the server of quota control actually

Reply:
Aha, ok. Then sorry. Keep it as it was.